importer/api/__init__.py

```diff
@@ -74,7 +74,7 @@ def import_course_from_path(path, repo_id, user_id):
     """
     bundle = XBundle()
     bundle.import_from_directory(path)
-    return import_course(bundle, repo_id, user_id)
+    import_course(bundle, repo_id, user_id)
```

Comment:
Okay, that makes sense. It would be trivial for me to create and push those tiny PRs while I'm working, and I'm sure they'd get pulled into `master` the same day, so same end result and cleaner.